packages/di/src/index.ts

```diff
@@ -4,7 +4,7 @@ import type { DependencyContainer } from "tsyringe";
 export type InstanceOf<T> = T extends new (...args: any[]) => infer R ? R : any;
 
 /**
- * It create on instance of a classe and store it globally using
+ * It create on instance of a classes and store it globally using
```

Comment:
Should be "It creates an instance of a class and stores it globally using"?

Reply:
![image](https://user-images.githubusercontent.com/43092101/154806097-b3ad17f4-647a-47f1-ae3d-2bdcd5c0eeb0.png)

Reply:
Why did you resolve this? What she said was correct.

Reply:
did not noticed

Reply:
refactored, review again.